src/controllers/userController.js

```diff
@@ -0,0 +1,153 @@
+import { userServices } from './../services/userService.js';
+import { v4 as uuidv4 } from 'uuid';
+
+let user = '';
+
+const defaultChatId = uuidv4();
+
+const rooms = [
+  {
+    id: defaultChatId,
+    creator: '',
+    name: 'Main-room',
+    history: [],
+  },
+];
+
+function messageValidation(message) {
+  if (!message || message.trim().length === 0) {
+    return 'Message required';
+  }
+
+  if (message.trim().length > 3000) {
+    return 'Message is to long';
+  }
+}
+
+function chatNameValidation(newChatName) {
+  if (!newChatName || newChatName.trim().length === 0) {
+    return 'Chat name required';
+  }
+
+  if (newChatName.trim().length < 2) {
+    return 'Chat name is too short';
+  }
+
+  if (newChatName.length > 30) {
+    return 'Less then 30 characters required';
+  }
+}
+
+const login = (req, res) => {
+  user = req.body.nickname;
+
+  res.send(200);
+};
+
+const setNewMessage = (req, res) => {
+  const { newMessage, roomId } = req.body;
+
+  if (messageValidation(newMessage)) {
+    return messageValidation(newMessage);
```

Comment:
The function `messageValidation` is called twice in a row. Consider storing the result in a variable and using it to avoid redundant calls.